diff --git a/internal/transport/transport_test.go b/internal/transport/transport_test.go
index 9f85005..19b50c2 100644
--- a/internal/transport/transport_test.go
+++ b/internal/transport/transport_test.go
@@ -1984,12 +1984,14 @@ func testFlowControlAccountCheck(t *testing.T, msgSize int, wc windowSizeConfig,
 		InitialWindowSize:     wc.serverStream,
 		InitialConnWindowSize: wc.serverConn,
 		StaticWindowSize:      true,
+		ReadBufferSize:        defaultReadBufSize,
 		BufferPool:            mem.DefaultBufferPool(),
 	}
 	co := ConnectOptions{
 		InitialWindowSize:     wc.clientStream,
 		InitialConnWindowSize: wc.clientConn,
 		StaticWindowSize:      true,
+		ReadBufferSize:        defaultReadBufSize,
 		BufferPool:            mem.DefaultBufferPool(),
 	}
 	server, client, cancel := setUpWithOptionsAndTimeout(t, 0, sc, pingpong, co, timeout)
